scripts/expectToBePx.ts

```diff
@@ -0,0 +1,53 @@
+import diff from 'jest-diff';
+
+expect.extend({
+  toBePx(received, expected) {
```

Comment:
I mean, I guess so? I'd just as much update the tests to have them match so we don't have to maintain this.

Reply:
My first pass updated all the tests with a regex replace, but then it turned out that not all of the internal functions seemed to be consistent with whether it used a number or a string with `px`. I suppose a helper function in the test file might be easier to maintain, though.

Reply:
I feel semi strongly to not take on this debt, but I'm curious about @DesignerDave's thoughts.

Reply:
```ts
function checkPixelValue(actual: string | number, expected: number) {
  if (typeof actual === 'string') {
    expect(actual).toBe(`${expected}px`);
  } else {
    expect(actual).toBe(expected);
  }
}
```

it's a pretty easy regex find/replace to just use this instead, so that's probably worth the small tradeoff in having very slightly worse diffs and error messages.

Reply:
Updated with the helper function